docs/src/content/docs/fr/manual-setup.mdx

```diff
@@ -62,20 +62,24 @@ Retrouvez toutes les options disponibles dans la [référence de configuration S
 
 ### Configurer les collections de contenu
 
-Starlight s'appuie sur les [collections de contenu d'Astro](https://docs.astro.build/fr/guides/content-collections/), qui sont configurées dans le fichier `src/content/config.ts`.
+Starlight s'appuie sur les [collections de contenu d'Astro](https://docs.astro.build/fr/guides/content-collections/), qui sont configurées dans le fichier `src/content.config.ts`.
 
-Créez ou mettez à jour le fichier de configuration du contenu, en ajoutant une collection `docs` qui utilise le schéma `docsSchema` de Starlight :
+Créez ou mettez à jour le fichier de configuration du contenu, en ajoutant une collection `docs` qui utilise les `docsLoader` et `docsSchema` de Starlight :
 
-```js ins={3,6}
-// src/content/config.ts
+```js ins={3-4,7}
+// src/content.config.ts
 import { defineCollection } from 'astro:content';
+import { docsLoader } from '@astrojs/starlight/loaders';
 import { docsSchema } from '@astrojs/starlight/schema';
 
 export const collections = {
-	docs: defineCollection({ schema: docsSchema() }),
+	docs: defineCollection({ loader: docsLoader(), schema: docsSchema() }),
 };
 ```
 
+Starlight supporte également [l'drapeau `legacy.collections`](https://docs.astro.build/fr/reference/legacy-flags/) où les collections sont gérées en utilisant l'implémentation de collections de contenu héritée.
```

Comment:
Is « l’drapeau » correct here? 👀 Never seen `l’` straight into a consonant before.

Reply:
Oh, excellent catch, that's entirely a mistake, I started with `le drapeau`, then switched to `l'option`, then switched back and switched back again after checking what translators picked on Astro Docs and I messed up the articles in the process 😆